Add timeout variable to attach and launch args (#528)
Microsoft/vscode-cordova: README.md

Due to recent changes in macOS we cannot connect to simulators to enable debugging of Cordova apps. We hope to re-introduce this functionality as soon as macOS provides a mechanism that allows the extension to connect to apps running in simulator. You can read more about the issue [here](https://github.com/google/ios-webkit-debug-proxy/issues/250).

> ### Error `'Cannot connect to runtime process, timeout after {0} ms - (reason: {1}).', '{_timeout}', '{_error}'` while running debug scenarios

Reviewer: ```suggestion
### Error `'Cannot connect to runtime process, timeout after {0} ms - (reason: {1}).', '{_timeout}', '{_error}'` occured while running debug scenarios
```

Try to increase attach timeout by adding `attachTimeout` configuration parameter.

> 

Reviewer: ```suggestion
```

> 

Author: ```suggestion
```

> This may happen if debugger is not being connected to the debug session in a default 10000 milliseconds. You can increase this time by adding `attachTimeout` variable to your debug scenario and initializing with the value that suits for your case.

Author: ```suggestion
This may happen if debugger is not being connected to the debug session in a default 10000 milliseconds. You can increase this time by adding `attachTimeout` variable to your debug configuration in `launch.json` and initializing with the value that suits for your case.
```

> This may happen if debugger is not being connected to the debug session in a default 10000 milliseconds. You can increase this time by adding `attachTimeout` variable to your debug scenario and initializing with the value that suits for your case.

Reviewer: ```suggestion
Try to increase attach timeout by adding `attachTimeout` configuration parameter.
```
